services/traefik/docker-compose.master.yml

```diff
@@ -1,4 +1,5 @@
-version: "3.7"
+version: "3.12"
```

Comment:
so just for info, these version top field should not be necessary anymore:
https://docs.docker.com/reference/compose-file/version-and-name/